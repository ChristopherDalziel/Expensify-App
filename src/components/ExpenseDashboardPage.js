import React from "react";

const ExpenseDashboardPage = () => {
  return <div>hello</div>;
};

export default ExpenseDashboardPage;
