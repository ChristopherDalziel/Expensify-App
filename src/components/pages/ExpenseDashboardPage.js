import React from "react";

const ExpenseDashboardPage = () => {
  return <div>Dashboard Page</div>;
};

export default ExpenseDashboardPage;
