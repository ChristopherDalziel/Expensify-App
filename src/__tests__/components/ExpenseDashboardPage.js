import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/pages/ExpenseDashboardPage";

import toJSON from "enzyme-to-json";

test("Should render dashboard page correctly", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
