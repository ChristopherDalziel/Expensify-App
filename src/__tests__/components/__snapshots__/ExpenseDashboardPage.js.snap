// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render dashboard page correctly 1`] = `
<Styled(div)>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</Styled(div)>
`;
